diff -urN meandmyshadow-0.2/CMakeLists.txt meandmyshadow-0.2-patched/CMakeLists.txt
--- meandmyshadow-0.2/CMakeLists.txt	2012-01-10 20:23:44.000000000 +1100
+++ meandmyshadow-0.2-patched/CMakeLists.txt	2012-01-15 20:10:11.801168281 +1100
@@ -77,7 +77,7 @@
     ${LibArchive_LIBRARY}
 )
 
-install(DIRECTORY ${PROJECT_BINARY_DIR}/data DESTINATION share/meandmyshadow/)
+install(DIRECTORY ${PROJECT_SOURCE_DIR}/data DESTINATION share/meandmyshadow/)
 install(TARGETS meandmyshadow RUNTIME DESTINATION bin)
 if("${CMAKE_SYSTEM_NAME}" MATCHES "Linux") 
 	install(FILES meandmyshadow.desktop DESTINATION share/applications/)
